src/probability/kernel/composition.lean

```diff
@@ -741,69 +741,72 @@ include mγ
 
 /-- Composition of two s-finite kernels. -/
 noncomputable
-def comp (η : kernel β γ) [is_s_finite_kernel η] (κ : kernel α β) [is_s_finite_kernel κ] :
-  kernel α γ :=
-snd (κ ⊗ₖ prod_mk_left α η)
+def comp (η : kernel β γ) (κ : kernel α β) : kernel α γ :=
```

Comment:
can you fix the file-level docstring, which says (twice) that the composition is defined for s-finite kernels?